src/Components/Web/src/PublicAPI.Unshipped.txt

```diff
@@ -13,3 +13,7 @@ Microsoft.AspNetCore.Components.Forms.InputText.Element.get -> Microsoft.AspNetC
 Microsoft.AspNetCore.Components.Forms.InputText.Element.set -> void
 Microsoft.AspNetCore.Components.Forms.InputTextArea.Element.get -> Microsoft.AspNetCore.Components.ElementReference?
 Microsoft.AspNetCore.Components.Forms.InputTextArea.Element.set -> void
+Microsoft.AspNetCore.Components.RenderTree.WebEventDescriptor.EventName.get -> string!
+Microsoft.AspNetCore.Components.RenderTree.WebEventDescriptor.EventName.set -> void
+*REMOVED*Microsoft.AspNetCore.Components.RenderTree.WebEventDescriptor.EventArgsType.get -> string!
+*REMOVED*Microsoft.AspNetCore.Components.RenderTree.WebEventDescriptor.EventArgsType.set -> void
```

Comment:
Although these look like public API changes, they are inside the `RenderTree` namespace that explicitly warns people not to use it. There has always been an analyzer that tells people that any uses of these types are subject to breaking changes.

Also, the `WebEventDescriptor.EventArgsType` should never have been used in any application code, since it's the protocol for dispatching events from JS to .NET, and user application code is not involved in that process.

Reply:
> Also, the WebEventDescriptor.EventArgsType should never have been used in any application code, since it's the protocol for dispatching events from JS to .NET, and user application code is not involved in that process.

Could you file an announcement for this? We didn't explicitly warn users against using it.

Reply:
Will do - https://github.com/dotnet/aspnetcore/issues/30275